templates/thank-you.html

```diff
@@ -46,24 +47,31 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
   {% endif %}
 
-  {% if content.view_submitted_response.enabled %}
+  {% if content.view_submitted_response.enabled and not content.view_submitted_response.expired %}
     <h1 data-qa="view-submitted-response-title">{{ _("Get a copy of your answers") }}</h1>
+    <p
+      class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
     {% if not content.view_submitted_response.expired %}
-      <p class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
-      {% call onsPanel({
-        "id": "view-submitted-response-warning",
-        "type": "warn"
-      }) %}
-        <p>{{ _("For security your answers will only be available to view for 45 minutes") }}</p>
-      {% endcall %}
-    {% else %}
-      {% call onsPanel({
+      {{ onsTimeoutPanel ({
+          "id": "view-submitted-response-countdown",
+          "minutesTextSingular": _("minute"),
+          "minutesTextPlural": _("minutes"),
+          "secondsTextSingular": _("second"),
+          "secondsTextPlural": _("seconds"),
+          "countdownText": _("For security, your answers will only be available to view for another"),
+          "nojsText": _("For security, your answers will only be available to view for another ") + content.view_submitted_response.expiry_minutes|string + _(" minutes"),
+          "redirectUrl":  url_for("post_submission.get_thank_you"),
+          "countdownExpiredText": _("For security, you can no longer view or get a copy of your answers"),
+          "sessionExpiresAt": content.view_submitted_response.expires_at
+      }) }}
+    {% endif %}
+  {% else %}
+    {% call onsPanel({
         "id": "view-submitted-response-guidance",
         "classes": "ons-u-mb-m"
       }) %}
-        {{ _("For security, you can no longer view your answers") }}
-      {% endcall %}
-    {% endif %}
+      {{ _("For security, you can no longer view or get a copy of your answers") }}
```

Comment:
This again looks like a duplication from above

Reply:
There was a bug which is fixed now. it is required when submitted response is enabled and session has expired. I have updated the condition